src/components/App.js

```diff
@@ -30,7 +30,7 @@ export default class App extends React.Component {
     return (
       <div className="container">
         <Display result={total} />
-        <ButtonPanel handleClick={(buttonName) => { this.handleClick(buttonName); }} />
+        <ButtonPanel clickHandler={(buttonName) => { this.handleClick(buttonName); }} />
```

Comment:
Instead of `clickHandler={(buttonName) => { this.handleClick(buttonName); }}`, find a way to make this shorter. You should be able to do something like:
```
<ButtonPanel clickHandler={this.handleClick} />
```